<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeRetestDialogBase</class>
 <widget class="QDialog" name="MergeRetestDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merge Test &amp; Retest data...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel2">
       <property name="maximumSize">
        <size>
         <width>32767</width>
         <height>32767</height>
        </size>
       </property>
       <property name="text">
        <string>Select test data files and Retest data (optional) to merge...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer30">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>12</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMergeFiles">
       <property name="toolTip">
        <string>Merge the files selected...</string>
       </property>
       <property name="text">
        <string>Merge Files Now!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabelFile1_3">
       <property name="text">
        <string>Merging mode:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxMergingMode">
       <property name="toolTip">
        <string>Mergiing mode: sorted by date, or order as found in dialog-box</string>
       </property>
       <item>
        <property name="text">
         <string>Rely on testing date (sort by date)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No sorting: Take files in listed order</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="spacer31">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>92</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabTestFiles">
      <attribute name="title">
       <string>Test Files</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="textLabelFile1">
         <property name="text">
          <string>Select Data files (Order ignored if &lt;i&gt;&lt;b&gt;'testing date'&lt;/b&gt;&lt;/i&gt; merging mode)</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="buttonSelectTestFile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select test data file</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/gex/icons/file_open.png</normaloff>:/gex/icons/file_open.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRemoveTestFile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Remove selected file from list</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/gex/icons/file_remove.png</normaloff>:/gex/icons/file_remove.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PushButtonClearAllTestFiles">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Remove ALL test files from list</string>
           </property>
           <property name="text">
            <string>Clear All</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer32">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>32</width>
             <height>9</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUpTest">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Move selection Up (Reorder)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/gex/icons/arrow_up.png</normaloff>:/gex/icons/arrow_up.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDownTest">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Move selection Down (Reorder)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/gex/icons/arrow_down.png</normaloff>:/gex/icons/arrow_down.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer33">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>180</width>
             <height>7</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="mUi_pQtwTestFilesView">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Test Data Files</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRetestFiles">
      <attribute name="title">
       <string>Retest Files (optional)</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="textLabelFile1_2">
         <property name="text">
          <string>Select &lt;b&gt;Retest&lt;/b&gt; Data files  (Order ignored if &lt;i&gt;&lt;b&gt;'testing date'&lt;/b&gt;&lt;/i&gt; merging mode)</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="buttonSelectRetestFile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select test data file</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/gex/icons/file_open.png</normaloff>:/gex/icons/file_open.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRemoveRetestFile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Remove selected file from list</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/gex/icons/file_remove.png</normaloff>:/gex/icons/file_remove.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PushButtonClearAllRetestFiles">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Remove ALL test files from list</string>
           </property>
           <property name="text">
            <string>Clear All</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer34">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>32</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUpRetest">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Move selection Up (Reorder)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/gex/icons/arrow_up.png</normaloff>:/gex/icons/arrow_up.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDownRetest">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Move selection Down (Reorder)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/gex/icons/arrow_down.png</normaloff>:/gex/icons/arrow_down.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer35">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>178</width>
             <height>12</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="mUi_pQtwRetestFilesView">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Retest Data Files</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
